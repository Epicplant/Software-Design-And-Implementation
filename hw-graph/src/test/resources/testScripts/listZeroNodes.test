# Tests that we correctly have a list of 0 nodes when ListNodes is called
# and the graph is empty

# Creates our new graph
CreateGraph newGraph

# Lists all of the nodes of this new graph
ListNodes newGraph