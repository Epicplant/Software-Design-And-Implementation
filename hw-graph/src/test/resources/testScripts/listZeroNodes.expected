# Tests that we correctly have a list of 0 nodes when ListNodes is called
# and the graph is empty

# Creates our new graph
created graph newGraph

# Lists all of the nodes of this new graph
newGraph contains: